### ISSUE TITLE
Primary monitor won't wake up from swayidle sleep

### ISSUE THREAD

**Issue by fuzunspm** [swaywm/sway#5604]
### Please fill out the following:
- **Sway Version:** v1.5

- **Debug Log:**
  [gist](https://gist.github.com/fuzunspm/522a93ba327c4ad74db5c2d80cea90e5)

- **Configuration File:**
[link](https://bin.privacytools.io/?5d04ac6d303fc485#nGIQ5/o1qhW+yvzk1nkH+PZXFf2r6BoFRDp+spYTqSM=)

- **Description:**

I don't know when this issue is started but it was working fine. Now, my primary monitor wakes up but every 2-3 seconds it loops itself with losing the image, no signal message on display and returning the image again

```
exec swayidle -w \
         timeout 600 'swaylock -f -c 000000' \
         timeout 900 'swaymsg "output * dpms off"' \
              resume 'swaymsg "output * dpms on"' \
         before-sleep 'swaylock -f -c 000000'
```
```
output DP-1 resolution 2560x1440 adaptive_sync on position 0,0 bg ~/Pictures/Wallpapers/IMG_4298.JPG center
output HDMI-A-1 resolution 3840x2160 scale 2 position 2560,0 transform 90 bg ~/Pictures/Wallpapers/wallhaven-639473.jpg fill
focus output DP-1
```

**Comment by Emantor** [comment]
Your debug log shows that sway is using the wayland backend, AFAIR this hasn't been tested much and relies on your underlying wayland compositor. Are you by chance using GDM or a similar login manager? If yes please try starting sway from a simple terminal session.

**Comment by fuzunspm** [comment]
Yes I'm using GDM. You are suggesting me to start sway from different DM ?

**Comment by panaman67** [comment]
@fuzunspm Sway does not officially support Desktop Managers (DM). Starting through a tty is the best option and works best for me. See the wiki for details
